src/main/java/edu/tamu/scholars/middleware/utility/DateFormatUtility.java

```diff
@@ -14,9 +14,17 @@ public class DateFormatUtility {
     private static final String[] datePatterns = {
         "yyyy",
         "E MMM dd HH:mm:ss z yyyy",
-        "yyyy-MM-dd'T'HH:mm:ss'Z'"
+        "yyyy-MM-dd'T'HH:mm:ss'Z'",
+        "dd-MM-yy",
+        "MM-dd-yyyy",
+        "yyyy-MM-dd HH:mm:ss",
+        "EEEEE MMMMM yyyy HH:mm:ss.SSSZ"
     };
 
+    public static String parseOutYear(String value) throws ParseException {
```

Comment:
Is this method used anywhere?
I do not see it used.

Is it through some sort of inheritance?

I think added unused code should be removed.

Reply:
Used in this PR, https://github.com/TAMULib/scholars-discovery/pull/283/files#diff-5a31d94a2050a565f694e09e47cda2f1ac9d03d638753ba6131a05a75036e209R258